src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_SIZE = 100;
+    private Random random = new Random();
+
+    public Figure getRandomFigure() {
+        ColorSupplier spl = new ColorSupplier();
```

Comment:
don't use short names, to fix it everywhere
ColorSupplier must be a class field
```suggestion
        ColorSupplier spl = new ColorSupplier();
```